src/shared/services/TopicService.js

```diff
@@ -0,0 +1,25 @@
+import TopicModel from '../models/TopicModel';
+export default class TopicService {
+    
+    //TODO: add tests when backend is "added" (called paths)
+    //temp
+    static topics = [
+        TopicModel.fromBackendData({ title: 'React Basic', describe: 'Lorem ipsum' }),
+        TopicModel.fromBackendData({ title: 'React Native Basic', describe: 'Lorem ipsum' }),
+        TopicModel.fromBackendData({ title: 'React + Redux', describe: 'Lorem ipsum' }),
+        TopicModel.fromBackendData({ title: 'React + Redux + Thunk', describe: 'Lorem ipsum' }),
+    ];
+    
+    static async getSubmittedTopics() {
+        return this.topics;
+    }
+    
+    static async getPastTopics() {
+        return this.topics;
```

Comment:
the same as `getSubmittedTopics`? Are you over engineering? 🙃

Reply:
@Tuhaj on design, we have `submitted topics` and some kind of `past workshops` which are quite a different things. There I put a different methods just for a sake of presence for future reference. Until    we actually have backend/firebase, it is kind of a dummy implementation. What is your take there?

Reply:
I get it, it's ok